protos/gimbal/gimbal.proto

```diff
@@ -17,6 +18,18 @@ message SetPitchAndYawResponse {
     GimbalResult gimbal_result = 1;
 }
 
+message SetModeRequest {
+    GimbalMode gimbal_mode = 1;
+}
+message SetModeResponse {
+    GimbalResult gimbal_result = 1;
+}
+
+enum GimbalMode {
+    FOLLOW = 0;
+    LOCK = 1;
```

Comment:
Please specify that this refers to YAW, so `YAW_FOLLOW` I'd say.

Reply:
Is that strictly reserved to yaw, on the gimbal side? Couldn't it follow with pitch as well?

Reply:
`YAW_FOLLOW` means that the gimbal follows the autopilot yaw, so it always looks forward (although with a little bit of damping so the picture is smoother).
`YAW_LOCK` means that the gimbal locks onto a specific heading no matter what direction the drone faces.

Pitch is basically always in `LOCK` mode for multicopters because you don't want it to be influenced by the vehicle pitch.